### DISCUSSION TITLE
specify a dir or file with `--typecheck` flag

### DISCUSSION THREAD

**Discussion by roccho-dev** [vitest-dev/vitest#4994] [Q&A]
Hi.
https://vitest.dev/guide/testing-types

With testing types,
can we specify a dir or file?

**Comment by hi-ogawa** [comment]
There is `typecheck.include` and this option supports glob. https://vitest.dev/config/#typecheck-include
For example, you can write something like `typecheck: { include: ["**/my-ts-dir/**"] }`. Would this work for you?

**Reply by roccho-dev** [reply]
@hi-ogawa 

Thanks.
Let me check what you tell me, will reply you later.
